packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -145,105 +144,91 @@ export interface LightningElementConstructor {
 
 export declare var LightningElement: LightningElementConstructor;
 
-export interface LightningElement {
-    // DOM - The good parts
-    dispatchEvent(event: Event): boolean;
-    addEventListener(
-        type: string,
-        listener: EventListener,
-        options?: boolean | AddEventListenerOptions
-    ): void;
-    removeEventListener(
-        type: string,
-        listener: EventListener,
-        options?: boolean | AddEventListenerOptions
-    ): void;
-    setAttributeNS(ns: string | null, attrName: string, _value: string): void;
-    removeAttributeNS(ns: string | null, attrName: string): void;
-    removeAttribute(attrName: string): void;
-    setAttribute(attrName: string, _value: string): void;
-    getAttribute(attrName: string): string | null;
-    getAttributeNS(ns: string, attrName: string): string | null;
-    getBoundingClientRect(): ClientRect;
-    querySelector<E extends Element = Element>(selectors: string): E | null;
-    querySelectorAll<E extends Element = Element>(selectors: string): NodeListOf<E>;
-    getElementsByTagName(tagNameOrWildCard: string): HTMLCollectionOf<Element>;
-    getElementsByClassName(names: string): HTMLCollectionOf<Element>;
-    classList: DOMTokenList;
-    isConnected: boolean;
-    toString(): string;
-
-    // LWC specifics
-    template: ShadowRoot;
-    render(): Template;
-
-    // Callbacks
-    connectedCallback?(): void;
-    disconnectedCallback?(): void;
-    renderedCallback?(): void;
-    errorCallback?(error: any, stack: string): void;
-
-    // Default HTML Properties
-    dir: string;
-    id: string;
-    accessKey: string;
-    title: string;
-    lang: string;
-    hidden: boolean;
-    draggable: boolean;
-    spellcheck: boolean;
-    tabIndex: number;
+type HTMLElementTheGoodParts = Pick<Object, 'toString'> &
+    Pick<EventTarget, 'dispatchEvent' | 'addEventListener' | 'removeEventListener'> &
```

Comment:
Instead of picking methods from the different interfaces, what' do you think of picking them from `HTMLElement` directly ?

Reply:
I don't mind either way, will make the update.